nomic/atlas.py

```diff
@@ -12,10 +12,144 @@
 from loguru import logger
 from tqdm import tqdm
 
-from .project import AtlasProject
+from .dataset import AtlasDataset
 from .settings import *
 from .utils import b64int, get_random_name, arrow_iterator
 
+from .data_inference import NomicProjectHyperparameters, NomicTopicHyperparameters, NomicDuplicatesHyperparameters
+
+
+def map_data(
+    data: List[Dict] = None,
```

Comment:
map_data should consume pandas df as well as list[Dict]. It looks cooler in examples, and pa.Table.from_pandas(df) tends to be better in every way (speed, memory, type safety) than pa.Table.from_pylist(lod).

Might as well consume arrow tables too, but not worth bragging about.

Reply:
done